docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,739 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    p = mp[:,0] #p: matrix profile (dist of each subseq to its NN)
</pre><p>Use <code>P = mp[:, 0]</code>  (uppercase)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
will be fixed in next commits.